pkg/virtctl/console/console.go

```diff
@@ -68,7 +68,7 @@ func (c *Console) Run(flags *flag.FlagSet) int {
 	}
 	vm := flags.Arg(1)
 
-	virtCli, err := kubecli.GetKubevirtClientFromFlags(server, kubeconfig)
+	virtCli, err := kubecli.GetKubevirtClientFromConfig(server, kubeconfig)
```

Comment:
I think that the solution for virtctl is to go with what I mentioned here: https://github.com/kubevirt/kubevirt/pull/648#pullrequestreview-89509254. That will make virtcl much more consistent with kubectl.